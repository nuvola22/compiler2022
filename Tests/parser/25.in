var i: integer = 0;
begin
while i < 10 do begin
i += 1;
writeln(i);
end;
end.