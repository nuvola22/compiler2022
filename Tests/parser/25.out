      program
         variable
            i
            integer
            0
         begin
            while
               <
                  i
                  10
               begin
                  +=
                     i
                     1
                  call
                     writeln
                     i
